torch/nn/functional.py

```diff
@@ -1861,12 +1881,12 @@ def affine_grid(theta, size):
     implement Spatial Transformer Networks.
 
     Args:
-        theta (Variable): input batch of affine matrices (N x 2 x 3)
-        size (torch.Size): the target output image size (N x C x H x W)
+        theta (Variable): input batch of affine matrices (:math:`N \times 2 \times 3`)
```

Comment:
I think with tensor/variable merge this should all be tensor?

Reply:
Might deserve a separate PR for that though

Reply:
I think there will a lot of places where `Variable` --> `Tensor`. May be for this PR, I could change it. Let me know what you think.